### ISSUE TITLE
How to pass data from the context of an activity to a child's workflow

### ISSUE THREAD

**Issue by zhangtingwz** [elsa-workflows/elsa-core#3324]
Hello, it's a pleasure to meet you, as well as using elsa workflow, in the process of using it, I encountered a problem：
![image](https://user-images.githubusercontent.com/26179617/194691202-0f0dac9a-e5e1-4c35-9045-4a6b053ceff4.png)
but 
![image](https://user-images.githubusercontent.com/26179617/194691246-c895c28d-e362-4c26-a24e-d92709cefe54.png)
please help me ，thank you very much

**Comment by zhangtingwz** [comment]
This is just an example, mainly to represent the values I want from the activity
![image](https://user-images.githubusercontent.com/26179617/194691628-32a79b8e-eb69-41f3-a25f-7bf4d23adc81.png)

**Comment by uppercuut** [comment]
this not an issue this is more like a question.. 

when you specify a `input `into the `RunWorkflow `method.  thats an input for the new workflow instance wide. 
so when you set a vibrable on the next child workflow. notice the `context`  is of type `ActivityExecutionContext`,
so its not workflow wide it's scoped within the activity you are on. so, if you want to get a variable that is  workflow wide use this `context.WorkflowInstance.Variables` i think you will fide you dto there as json. 

and if u have more question here is the discord link for that 
https://discord.gg/F5tCv2bs

cheers

**Comment by zhangtingwz** [comment]
Yes, my friend, this is really a problem. Because of the wall, I can't go to Stack Overflow. Then elsa is a new workflow, and there are fewer people using it. So I can only go to github to find the answer. I'm sorry for causing you trouble, and thank you for your help
